Static build (#1283)
ROCm/hipBLASLt: docs/installation.rst

|                                           | directory, don`t use ``-i`` flag.  |
+-------------------------------------------+------------------------------------+

Static Library
----------------

> To build static libraries with ``install.sh`` use ``--static``.
> This produces a non-standard static library build. That is, there is an additional runtime dependency 
> which is the entire subdirectory ``hipblaslt/`` located in the ``/opt/rocm/lib`` folder. 
> One can relocate this folder, but the environment variable ``HIPBLASLT_TENSILE_LIBPATH`` must be set
> accordingly.

Reviewer: ```suggestion
To build static libraries with ``install.sh``, use ``--static``.
This produces a non-standard static library build. This means it has an additional runtime dependency 
consisting of the entire ``hipblaslt/`` subdirectory, which is located in the ``/opt/rocm/lib`` folder. 
You can relocate this folder, but you must set the environment variable ``HIPBLASLT_TENSILE_LIBPATH``
accordingly.
```
